Sources/ParseSwift/Objects/ParseObject+updatable.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  ParseObject+updatable.swift
```

Comment:
Can you change this to a more applicable name. It's not attached to a ParseObject, so it doesn't need this in the file name. It can be called something like `ParseMutable` and can be moved into this folder: https://github.com/parse-community/Parse-Swift/tree/main/Sources/ParseSwift/Protocols